pkg/pluginsupplier/plugin_supplier_test.go

```diff
@@ -141,222 +130,6 @@ var _ = Describe("GetInstalledPlugins (standalone plugins)", func() {
 			Expect(isInstalled).To(BeFalse())
 		})
 	})
-
-	Context("when a standalone and server plugin for k8s target installed", func() {
```

Comment:
Since the `MigrateContextPluginsAsStandaloneIfNeeded()` function is no longer called by `pluginsupplier.GetInstalledPlugins()`, these tests were no longer applicable.

I however moved them to a new `cleanup_test.go` file to directly test `MigrateContextPluginsAsStandaloneIfNeeded()`